addons/sale_pdf_quote_builder/tests/test_pdf_quote_builder.py

```diff
@@ -248,6 +248,32 @@ def test_quotation_document_upload_for_template(self):
             "Quotation document shouldn't have a company id",
         )
 
+    def test_quotation_document_is_removed_on_template_change(self):
+        so_tmpl = self.env['sale.order.template'].create({
+            'name': "test1",
+            'quotation_document_ids': [
+                Command.link(self.header.id),
+            ],
+        })
+        so_tmpl_2 = self.env['sale.order.template'].create({
+            'name': "test2",
+        })
```

Comment:
```suggestion
        so_tmpl_2 = self.env['sale.order.template'].create({'name': "test2"})
```
To be applied everywhere, if it fits in a single line (<100chars), good enough, no need to overly extend it

Reply:
Same here, you can check the other lines in your test